test/integration/connect/envoy/test-envoy-versions.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
This is just a script to assist with running the current cases through a version matrix to deduce envoy versions for plugging into those version checks.